main/src/modules/reactiveLayer/behaviorManager/include/pushingRight.h

```diff
@@ -0,0 +1,35 @@
+#ifndef PUSHINGRIGHT
```

Comment:
I have no any idea about this problem! I introduced these new methods with UPF request to connect `behaviorManager` and `Karma`, so please leave me out of these discussion!
When you guys finish with solution to integrate, let me know!